splunk-otel-android-volley/README.md

```diff
@@ -0,0 +1,59 @@
+
+# Splunk OpenTelemetry Android Volley Instrumentation
+
+> :construction: &nbsp;Status: Experimental
+
+This directory contains the Splunk instrumentation for the [Volley](https://google.github.io/volley/)
+HTTP client library. If you use the Volley HTTP client in your Android application, you can
+leverage this instrumentation to help simplify tracing and capture of HTTP headers.
+
+## Add to your project
+
+In addition to the [main Splunk Android SDK dependency](https://github.com/signalfx/splunk-otel-android#getting-the-library),
+you will also need to add the `splunk-otel-android-volley` dependency to your `build.gradle.kts`:
+
+
+```gradle
+dependencies {
+    ...
+    implementation("com.splunk:splunk-otel-android-volley:0.16.0")
+    ...
+}
+```
+
+## How to use VolleyTracing
+
+See the class `VolleyExample.java` file in the sample application in this repository for an
+example of how to create an instrumented client HTTP call.
+
+First, you will create an instance of `VolleyTracing` by passing your `splunkRum` instance
+to the builder:
+
+```java
+VolleyTracing volleyTracing = VolleyTracing.builder(splunkRum).build();
+```
+You can also add extra request/response headers to capture to the builder.
+These headers will be attached to the RUM Volley HTTP client spans with
+a`http.response.header.` prefix.
```

Comment:
Prefix is given only for response headers although previous sentence mentions both request and response headers. There should be a space between a`http.response.header.`

Reply:
Good catch. Thanks.